projects/plugins/jetpack/_inc/client/dashboard-widget/style.scss

```diff
@@ -1,3 +1,15 @@
+@mixin widget-card-background-container() {
+	background: #F9F9F6;
```

Comment:
Maybe we can use variables in here, like `var( --jp-white-off )` in this case?

Reply:
Done! I slightly refactored the `base-styles` package to bring in those CSS variables without extraneous styling.